publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -276,6 +298,10 @@ private void initService() {
         if (channelsCount == null ) {
             channelsCount = Integer.getInteger(getValuesFromSystemProperties(protocol + ".rabbitmq.channelsCount"));
         }
+
+        if (protocol.equalsIgnoreCase("eiffelsemantics") && routingkeyTypeOverrideFilePath == null) {
+            routingkeyTypeOverrideFilePath = getValuesFromSystemProperties("semanticsRoutingkeyTypeOverrideFilepath");
```

Comment:
Should PropertiesConfig.SEMANTICS_ROUTINGKEY_TYPE_OVERRIDE_FILEPATH be used instead of "semanticsRoutingkeyTypeOverrideFilepath"?

Reply:
Modified